Mathlib/CategoryTheory/Filtered/Final.lean

```diff
@@ -310,4 +311,20 @@ theorem Functor.initial_of_isCofiltered_pUnit [IsCofiltered C] (F : C ⥤ Discre
   · use c; use 𝟙 c
     apply Subsingleton.elim
 
+/-- The functor `StructuredArrow.proj : StructuredArrow Y T ⥤ C` is final if `T : C ⥤ D` is final
+and `C` is filtered. -/
+instance StructuredArrow.final_proj_of_filtered [IsFilteredOrEmpty C]
```

Comment:
```suggestion
instance StructuredArrow.final_proj_of_isFiltered [IsFilteredOrEmpty C]
```
or
```suggestion
instance StructuredArrow.final_proj_of_isFilteredOrEmpty [IsFilteredOrEmpty C]
```

Reply:
Keeping quiet about the `OrEmpty` part seems more in line with the rest of the file